src/main/java/se/sundsvall/casestatus/integration/db/model/BaseEntity.java

```diff
@@ -0,0 +1,76 @@
+package se.sundsvall.casestatus.integration.db.model;
+
+import java.util.Objects;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Id;
+import jakarta.persistence.MappedSuperclass;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.experimental.SuperBuilder;
+
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@MappedSuperclass
+@SuperBuilder(setterPrefix = "with")
+public abstract class BaseEntity {
+
+	@Id
+	@Column(name = "flow_instance_id")
+	private String flowInstanceId;
+
+	@Column(name = "family_id")
+	private String familyId;
+
+	@Column(name = "status")
+	private String status;
+
+	@Column(name = "errand_type")
+	private String errandType;
+
+	@Column(name = "content_type")
+	private String contentType;
+
+	@Column(name = "first_submitted")
+	private String firstSubmitted;
+
+	@Column(name = "last_status_change")
+	private String lastStatusChange;
+
+	@Column(name = "municipality_id")
+	private String municipalityId;
```

Comment:
Add index on this column (in subcklasses)?

Reply:
Maybe add index on other columns used in queries as well? It seems that indicies aren't used at all in this db?